country_limiter.go

```diff
@@ -13,8 +13,9 @@ import (
 )
 
 type CountryLimiter struct {
-	db        *maxminddb.Reader
-	countries []string
+	db                         *maxminddb.Reader
+	limitRateForOtherCountries bool
```

Comment:
Just an idea, how about specifying the type?

```suggestion
	CondType CondType
```

```go
type CondType bool
var Target CondType = true
var Excect CondType = false
```